images.go

```diff
@@ -202,8 +202,8 @@ var Images = []Image{
 		Name: "golang",
 		Tags: []Tag{
 			Tag{
-				Sha: "a325153f1f1e2edb76a76ad789aff172b89dd6178e8f74d39ef87c04d87d1961",
-				Tag: "1.10.4",
+				Sha: "63ec0e29aeba39c0fe2fc6551c9ca7fa16ddf95394d77ccee75bc7062526a96c",
```

Comment:
Aren't we just adding new tag usually? So that if there are references somewhere - we will not break it. Or is that a purpose - break all the builds and force them to update tag?

Reply:
I was not sure how this work, I though we needed to add the tag and sha from the target image according to readme: https://github.com/giantswarm/retagger#tags.

Reply:
reverted sha, PTAL